src/utils/debounce.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Ghostery Browser Extension
+ * https://www.ghostery.com/
+ *
+ * Copyright 2017-present Ghostery GmbH. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0
+ */
+
+export default function debounce(fn, { waitFor, maxWait }) {
```

Comment:
To not introduce not needed changes, I did not use this new util in `content_scripts/adblocker.js` on purpose. I'll make another PR when this is merged.